### PR TITLE
:wrench: build script -> rest/spread properties

### PR DESCRIPTION
**Pull request by fabienjuif**
Drawbacks : 
 - Build time
 - Slow startup

What to improve (in an other PR) : 
 - Have a specific babelrc to transpile src/script so it doesn't transpile async/await by example and the startup could be faster

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,15 +10,18 @@
     "start": "./src/script/index.js",
```
Comment (bpetetot) [inline package.json]: now it's `"start": "index.js",`

File: package.json [hunk 2]
```diff
@@ -10,15 +10,18 @@
...
     "test": "jest",
     "coveralls": "jest --coverage && cat ./coverage/lcov.info | coveralls",
-    "build:prod": "cross-env NODE_ENV=build rollup -c ./misc/rollup.config.js",
-    "build:dev": "npm run build:prod -- -w",
-    "build": "npm run build:prod"
+    "build:gui:prod": "cross-env NODE_ENV=build rollup -c ./misc/rollup.gui.config.js",
+    "build:gui:dev": "npm run build:gui:prod -- -w",
+    "build:script:prod": "cross-env NODE_ENV=build rollup -c ./misc/rollup.script.config.js",
+    "build:script:dev": "npm run build:script:prod -- -w",
```
Comment (bpetetot) [inline package.json:16]: I've got a build error in watch mode for the script.
When I want to update a file, I have this error : 
```
🚨   Could not load fs (imported by /Users/zuhmilo/workspaces/workspace-perso/react-workbench/src/script/config/bluebird.js): ENOENT: no such file or
 directory, open 'fs'
```

I think we need a [rollup plugin](https://github.com/calvinmetcalf/rollup-plugin-node-builtins), or put the platform target in babel configuration to node

**Review by bpetetot — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (fabienjuif) [reply package.json:16]: Weird I didn't have the problem.
I'll look into it

**Comment by fabienjuif** [comment]
@bpetetot I let you try the `build:script:dev` since I can't reproduce it

**Comment by bpetetot** [comment]
@fabienjuif 
I've got a different error
```
🚨   Could not load react-docgen/dist/utils (imported by /Users/zuhmilo/workspaces/workspace-perso/react-workbench/src/script/server/api/services/doc
gen/resolvers/componentResolver.js): ENOENT: no such file or directory, open 'react-docgen/dist/utils'
```

Did you try to modify a file with watching mode enabled ?

**Comment by fabienjuif** [comment]
Poop PR, new one is #93

**Pull request closed without merging**